src/main/kotlin/br/unipar/plano/domain/reembolso/model/Carteirinha.kt

```diff
@@ -0,0 +1,27 @@
+package br.unipar.plano.domain.reembolso.model
+
+import java.time.LocalDate
+import java.util.*
+import javax.persistence.*
+
+enum class StatusCarteirinha {
+    ATIVA, VENCIDA
+}
+
+@Entity
+class Carteirinha (
+    @Id
+    val id: UUID,
+
+    @Column(nullable = false)
+    val numero: String,
+
+    @Enumerated(EnumType.STRING)
+    val statusCarteirinha: StatusCarteirinha = StatusCarteirinha.ATIVA,
+
+    @Column(nullable = false)
+    val dataValidade: LocalDate,
+
+    @OneToOne(cascade = [CascadeType.ALL])
```

Comment:
O usuário já pode ter solicitadas diversas carterinhas, seja por vencimento ou extravio.

```suggestion
    @ManyToOne(cascade = [CascadeType.ALL])
```